var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -19,6 +19,8 @@ class Mumps(Package):
     variant('float', default=True, description='Activate the compilation of smumps')
     variant('complex', default=True, description='Activate the compilation of cmumps and/or zmumps')
     variant('idx64', default=False, description='Use int64_t/integer*8 as default index type')
+    variant('tests', default=False, description='Run small MPI unit tests')
```

Comment:
@alalazo well, here it's intentionally false as it (i) does not work for me with MPI; (ii) if installed on cluster frontends, mpirun may not be available.